phpClient/home.html

```diff
@@ -4,7 +4,10 @@
 <head>
     <title>Subless PHP sample</title>
     <!-- Load subless.js onto pages with content owned by a particular creator to automatically track a user -->
-    <script src="https://pay.subless.com/dist/subless.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/oidc-client/1.11.5/oidc-client.js"
```

Comment:
Maybe a little explainer about where/why the oidc-client is coming from might be good, but it's ndb.